<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertScriptDialog</class>
 <widget class="QDialog" name="InsertScriptDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="pteScript"/>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="cbUseTable">
       <property name="text">
        <string>Use table</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbUpdatePointers">
       <property name="text">
        <string>Update pointers</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="rbLayout">
       <item>
        <widget class="QRadioButton" name="rbLittleEndian">
         <property name="text">
          <string>Little-endian</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbBigEndian">
         <property name="text">
          <string>Big-endian</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <widget class="QDialogButtonBox" name="bbControls">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bbControls</sender>
   <signal>accepted()</signal>
   <receiver>InsertScriptDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbControls</sender>
   <signal>rejected()</signal>
   <receiver>InsertScriptDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
